assets/scripts/segments/drag_and_drop.js

```diff
@@ -1,55 +1,40 @@
 import { trackEvent } from '../app/event_tracking'
 import { loseAnyFocus } from '../util/focus'
-import { getStreetSectionTop } from '../app/window_resize'
 import { infoBubble } from '../info_bubble/info_bubble'
 import { app } from '../preinit/app_settings'
-import { system } from '../preinit/system_capabilities'
 import { setIgnoreStreetChanges } from '../streets/data_model'
 import { getElAbsolutePos } from '../util/helpers'
 import { generateRandSeed } from '../util/random'
-import { BUILDING_SPACE } from './buildings'
 import {
   SegmentTypes,
   getSegmentInfo,
   getSegmentVariantInfo
 } from './info'
 import {
-  SHORT_DELAY,
```

Comment:
Since this is no longer imported, also make it not exported in the original file.